labonneboite/tests/web/api/test_api_base.py

```diff
@@ -489,14 +490,15 @@ def setUp(self, *args, **kwargs):
                 raise ValueError("Cannot create an entry in Office with a city absent from self.positions.")
 
             office = Office(
-                company_name=doc['name'],
+                office_name=doc['name'],
                 siret=doc['siret'],
                 score=doc['score'],
                 naf=doc['naf'],
                 city_code=commune_id,
                 zipcode=zip_code,
                 email=u'foo@bar.com',
                 departement=zip_code[:2],
+                company_name=doc['company_name'] if 'company_name' in doc.keys() else '',
```

Comment:
`doc.get('company_name', '')`

De manière générale, il ne faut jamais écrire `x in d.keys()`, qui correspond à une recherche dans une liste, mais plutôt `x in d`.

Reply:
In fact...